data/borderlines.json

```diff
@@ -351,6 +305,52 @@
           "latitude":85.3273
         }
       ]
+  },
```

Comment:
Format the code using prettier.

Reply:
updated with suggestion, added indention in front